test/Reader/IniTest.php

```diff
@@ -95,4 +95,26 @@ public function testFromStringNested()
         $this->assertEquals($arrayIni['bla']['foo']['baz'][0], 'foobaz1');
         $this->assertEquals($arrayIni['bla']['foo']['baz'][1], 'foobaz2');
     }
+
+    public function testFromFileParseSections()
+    {
+        $arrayIni = $this->reader->fromFile($this->getTestAssetPath('sections'));
```

Comment:
as I can see we have tests just for `fromFile`, but we should add the same also for `fromString`

Reply:
Done